pkg/actuators/machine/awsclient.go

```diff
@@ -14,7 +12,7 @@ type AwsClientWrapper struct {
 	client awsclient.Client
 }
 
-var _ types.CloudProviderClient = &AwsClientWrapper{}
+//var _ types.CloudProviderClient = &AwsClientWrapper{}
```

Comment:
What's the deal here? Do we not implement this interface anymore? If so, we should just delete this. If not, we should un-comment it.

Reply:
no, we don't, since this was a left over of when we vendored the e2e tests. Removed.